dev/com.ibm.ws.sipcontainer/src/com/ibm/ws/sip/stack/transport/sip/netty/SipTlsOutboundChannelFactory.java

```diff
@@ -0,0 +1,52 @@
+/*******************************************************************************
```

Comment:
Are we making use of this?

Reply:
yes, referred to by class in GenericChannelProvider